mod/item.php

```diff
@@ -58,12 +58,12 @@ function item_post(App $a) {
 	// logger('postinput ' . file_get_contents('php://input'));
 	logger('postvars ' . print_r($_REQUEST,true), LOGGER_DATA);
 
-	$api_source = ((x($_REQUEST, 'api_source') && $_REQUEST['api_source']) ? true : false);
+	$api_source = x($_REQUEST, 'api_source') && $_REQUEST['api_source'];
```

Comment:
You can simplify this further to `$api_source = (x($_REQUEST, 'api_source'));` as `x()` already check for truthiness if the value exists.